tests/consumer_tests/routers/test_inventory.py

```diff
@@ -94,7 +114,7 @@ def test_serialize_device_required_params(key, response_row) -> None:
     del response_row[key]
 
     with pytest.raises(KeyError):
```

Comment:
Could consider using `@pytest.mark.xfail(raises=KeyError, strict=True)`, but up to you!